code/modules/atmospherics/machinery/other/refill_station.dm

```diff
@@ -0,0 +1,200 @@
+/// The maximum `target_pressure` you can set on the station. Equates to about 1013.25 kPa.
+#define MAX_TARGET_PRESSURE 10 * ONE_ATMOSPHERE
+
+/obj/machinery/atmospherics/refill_station
+	name = "refill station"
+	icon = 'icons/obj/atmos.dmi'
+	icon_state = "filler_oxy"
+	anchored = TRUE
+	density = TRUE
+	resistance_flags = NONE
+	idle_power_consumption = 1
+	active_power_consumption = 5
+	power_channel = PW_CHANNEL_ENVIRONMENT
+	/// The desired pressure the refill station should be outputting into a holding tank.
+	target_pressure = MAX_TARGET_PRESSURE
+	can_unwrench_while_on = FALSE
+	/// The current air contents of this device
+	var/datum/gas_mixture/air_contents = new
+	/// The tank inserted into the machine
+	var/obj/item/tank/holding_tank
+	/// The maximum pressure of the device
+	var/maximum_pressure = 10 * ONE_ATMOSPHERE
+
+/obj/machinery/atmospherics/refill_station/Initialize(mapload)
+	. = ..()
+	SSair.atmos_machinery += src
+	air_contents.volume = 1000
+	air_contents.set_temperature(T20C)
+
+/obj/machinery/atmospherics/refill_station/examine(mob/user)
+	. = ..()
+	. += "The Nanotrasen standard [src] is a vital piece of equipment for \
```

Comment:
![imagen](https://github.com/user-attachments/assets/ac5cfc05-ac88-4986-bdb9-6396aca4e414)

Refill station's examine text has an extra "the" in it.


Apart from this, all stations refer to "various mixes". It would be better if each station mentioned its own mix, or the description could also talk about refill stations in general - instead of the specific type.